cmake/IgnPackaging.cmake

```diff
@@ -148,6 +148,12 @@ macro(_gz_setup_packages)
 endmacro()
 
 macro(ign_create_packages)
+  # TODO(chapulina) Enable warnings after all libraries have migrated.
+  # message(WARNING "ign_create_packages is deprecated, use gz_create_packages instead.")
+
+  gz_create_packages()
+endmacro()
+macro(gz_create_packages)
 
   #============================================================================
   # Load platform-specific build hooks if present.
```

Comment:
I'm unsure if args need to be bubbled down here to be picked up by `_gz_load_build_hooks`?

Reply:
Edit: Scratch that, it should be fine because we're using macros